source/client/cl_presence.c

```diff
@@ -501,40 +501,51 @@ typedef struct RichPresence {
 } RichPresence;
 
 typedef struct {
-	_Bool initialized;
+	_Bool discord_initialized;
 	unsigned int next_update;
 	RichPresence old_presence;
 } cl_discord_state_t;
 
-static cl_discord_state_t cl_discord_state;
+static cl_discord_state_t cl_presence_state;
```

Comment:
Might as well rename the entire struct then?

Reply:
good catch, forgot about that

Reply:
https://github.com/CoolElectronics/warfork-qfusion/commit/3d3924b11b5056bf55f3ad75454589040df2ae0d